docs/pipeline.rst

```diff
@@ -173,6 +173,18 @@ intent_classifier_sklearn
     to other classifiers it also provides rankings of the labels that did not "win". The spacy intent classifier
     needs to be preceded by a featurizer in the pipeline. This featurizer creates the features used for the classification.
 
+intent_featurizer_regex
```

Comment:
not something we have to resolve right now, but with this change featurizers are no longer strictly `intent_featurizers`